api/db-cdr/generate-cdr/bq-schemas/achilles_analysis.json

```diff
@@ -0,0 +1,40 @@
+[
```

Comment:
Long-term I think it'd be better to pull as much of these from curation as possible... when there's a new CDR release, the  schema can change, and it's better not to have to worry about keeping this in sync.

For now, seems fine.

Reply:
I thought of that. Fine except if there are breaking changes .. I think on principle workbench should be in charge of its own schemas.  I agree, fine for now.